app/javascript/components/applications/application_form/preferences/LiveWorkFields.js

```diff
@@ -43,6 +43,7 @@ const LiveWorkFields = ({ i, householdMembers, individualPreference }) => {
           fieldName={buildFieldId(i, 'individual_preference')}
           options={individualPreferenceOptions}
           validation={validate.isPresent('Individual Preference is required')}
+          onChange={() => form.change(`preferences[${i}].type_of_proof`, '')}
```

Comment:
Here we trigger the reset of the `type_of_proof` on change since the 2 values have 2 entirely different lists of possible options